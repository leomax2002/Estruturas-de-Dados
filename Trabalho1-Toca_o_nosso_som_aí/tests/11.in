add YellowSubmarine
play
add Help
ended
current
fight