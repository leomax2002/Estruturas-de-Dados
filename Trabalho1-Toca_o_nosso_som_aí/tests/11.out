Help
Jedi Wagner, assuma o comando!